mmdet3d/datasets/lyft_dataset.py

```diff
@@ -335,9 +335,15 @@ def format_results(self, results, jsonfile_prefix=None, csv_savepath=None):
         else:
             tmp_dir = None
 
-        if not isinstance(results[0], dict):
+        # this is a bit hack to enable testing KITTI detectors on nuScenes
```

Comment:
It's not necessarily KITTI detectors. The issue just takes those models as examples. We can just add a general comment here to indicate the expected format of results.

Reply:
Also please note that here should be Lyft instead of nuScenes

Reply:
I am not sure if my modification is acceptable now. Please kindly check it.